examples/account_management/link_manager_to_client.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-# Copyright 2019 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
This should probably stay the same, it's the date the file was created, not when it was edited. This file looks like it was created in June 2019.

Reply:
+1. We leave the copyright at file creation year.

Reply:
Right, I think I had a keyboard-cat incident and wrote in 2020 instead of undoing. Reverted.